CareKit/SymptomTracker/OCKSymptomTrackerViewController.m

```diff
@@ -84,8 +84,7 @@ - (void)viewDidLoad {
                                                                               style:UIBarButtonItemStylePlain
                                                                              target:self
                                                                              action:@selector(showToday:)];
-    self.navigationItem.rightBarButtonItem.tintColor = self.progressRingTintColor;
-    
```

Comment:
The developer can provide a progressRingTintColor that is then applied to the ring and the `rightBarButtonItem`. We want this for consistency in the color scheme. If the developer chooses to use a different color than the progress ring for the `rightBarButtonItem`, then they can just specify it on the navigation controller.

We probably don't want to remove this functionally.

This behavior is also mimicked in `OCKCareCardViewController`.